app/Http/MailChimpClient.php

```diff
@@ -198,6 +201,11 @@ public function putSubscriber( array $mcData, string $email = null ) {
 				throw new InvalidEmailException( $put['errors'][0]['message'] );
 			}
 		}
+		if ( isset( $put['status'] ) && is_numeric( $put['status'] ) && $put['status'] !== 200 ) {
+			if ( isset( $put['detail'] ) && strpos( $put['detail'], 'compliance state' ) ) {
```

Comment:
Ich nehme an, die Warnung in https://www.php.net/manual/en/function.strpos.php  ist hier nicht relevant (z.B. dass 'compliance state' zu Beginn steht und dann der Index 0 wird, was False wird).

Entsprechend gleiche Fragen für MemberDeleteException weiter unten.

Reply:
Genau. Der vollständige `detail` Wert lautet 'mail@example is in a compliance state due to unsubscribe, bounce or compliance review and cannot be subscribed.' entsprechend gibt `strpos` einen Wert > 0 zurück und ist somit thruthy.